file_upload_handler/src/server.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express'
+import multer from 'multer'
+
+const app =express();
+
+const storage = multer.diskStorage({
+    destination: function (req, file, cb) {
+      cb(null, './uploads')
+    },
+    filename: function (req, file, cb) {
+      const uniquepreffix = Date.now() + '-' + Math.round(Math.random() * 1E9)
+      cb(null,uniquepreffix + "-"+ file.originalname )
+    }
+  })
+
+  function fileFilter (req, file, cb) {
+
+    // The function should call `cb` with a boolean
+    // to indicate if the file should be accepted
+    const acceptFileType = ['video/mp4','image/png','image/jpeg'  ,'image/gif']
+    if (acceptFileType.includes(file.mimetype )){
+        // To accept the file pass `true`, like so:
+        cb(null, true)
+    }else{
+        cb(new Error("invalid File type\nonly accpet" + acceptFileType ))
+    }
+    
+    //cb(null, false)
+  
+    
+  
+    // You can always pass an error if something goes wrong:
+    //cb(new Error('I don\'t have a clue!'))
```

Comment:
Should remove commented out code

Reply:
There is an option to add and commit individual lines of code, rather than the whole file, you might want to look into that